packages/app-project/src/screens/ClassifyPage/ClassifyPageContainer.spec.js

```diff
@@ -410,7 +422,24 @@ describe('Component > ClassifyPageContainer', function () {
       })
 
       expect(wrapper.find(ClassifyPage).props().workflowFromUrl).to.equal(workflows[1])
-      routerStub.restore()
+      expect(wrapper.find(ClassifyPage).props().workflowID).to.equal(workflows[1].id)
+    })
+
+    it('should be able to load a completed workflow from the url', function () {
+      // note the workflow prop is an empty array, which is what fetchWorkflowHelper would return if the workflow is complete
```

Comment:
I don’t think this is true on projects like Davy Notebooks, which have more than one workflow.

Reply:
Yeah, the workflows array will contain all the incomplete project workflows plus the specified workflow even if it’s complete. 

https://github.com/zooniverse/front-end-monorepo/blob/3e363a4a899f0e1fe0942274d4e6232e906df91d/packages/app-project/src/helpers/fetchWorkflowsHelper/fetchWorkflowsHelper.js#L103-L108

Reply:
Workflow assignment projects have multiple workflows. If the level 1 workflow completes, then the first workflow in the array of active, incomplete workflows will now be level 2.

Reply:
I think with workflow assignment enabled, if tried to access complete level 1 workflow it would load but show complete banner. if tried to load level 2 (and don't have access) it'd prompt with workflow selection modal, so user would be stuck (because level 1 wouldn't show). could theoretically be an issue, but hopefully projects using workflow assignment know to always have lower level workflows active/incomplete.